src/structures/CommandInteraction.js

```diff
@@ -187,6 +187,24 @@ class CommandInteraction extends Interaction {
    * @property {Role|Object} [role] The resolved role
    */
 
+  /**
+   * Send a followup message to this interaction.
+   * @param {string|APIMessage|MessageAdditions} content The content for the reply
+   * @param {InteractionReplyOptions} [options] Additional options for the reply
+   * @returns {Promise<Message|Object>}
+   */
+  async followup(content, options) {
```

Comment:
According to [Grammarly](https://www.grammarly.com/blog/follow-up-followup/), this is wrong, the verb is to `follow up` and the act is a `follow-up`, both of which are two words. As such, this should be renamed to `followUp` following camelCase conventions.

![image](https://user-images.githubusercontent.com/24852502/118092452-6b4f6e00-b3cc-11eb-9a70-1b928d382c20.png)

```suggestion
  async followUp(content, options) {
```

Reply:
I agree 100% buuuuuut

![Screenshot_20210513-182414_Chrome](https://user-images.githubusercontent.com/5294381/118099415-8942c000-b418-11eb-819e-087c7647ffa9.jpg)

Reply:
That deserves a PR on their end (plus, you know how many typos people find in the discord-api-docs repo).